src/metabase/driver/postgres.clj

```diff
@@ -280,7 +289,7 @@
 (defmethod sql.qp/date [:postgres :week-of-year-iso] [_driver _ expr] (extract-integer :week expr))
 
 (defmethod sql.qp/date [:postgres :day-of-week]
-  [_ driver expr]
+  [driver _unit expr]
```

Comment:
these were out of order, luckily it still worked because it was falling back to the default impl. But if we wrote a `:day-of-week` driver this would have been a bug